boot/zephyr/Kconfig

```diff
@@ -8,7 +8,7 @@ mainmenu "MCUboot configuration"
 comment "MCUboot-specific configuration options"
 
 # Hidden option to mark a project as MCUboot
-config MCUBOOT
+config IS_MCUBOOT
```

Comment:
```suggestion
# Hidden option to mark a project as MCUboot
config IMCUBOOT
  select IS_MCUBOOT
```

Reply:
By redefinition, do you mean re-naming? If so I understand, though this one entry has just been introduced so I don't see a big issue.